resources/model/OpenStudio.idd

```diff
@@ -16663,11 +16668,41 @@ OS:DistrictHeating,
        \required-field
        \type object-list
        \object-list ConnectionNames
-  N1 ; \field Nominal Capacity
+  N1 , \field Nominal Capacity
        \units W
        \minimum 0.0
        \autosizable
        \required-field
+  A5 ; \field Capacity Fraction Schedule
+       \note Schedule values are multiplied by Nominal Capacity for current capacity
+       \type object-list
+       \object-list ScheduleNames
+
+OS:DistrictHeating:Steam,
+       \memo Centralized source of Steam, such as a district heating system.
+  A1,  \field Handle
+       \type handle
+       \required-field
+  A2 , \field Name
+       \required-field
+       \reference ConnectionObject
+  A3 , \field Steam Inlet Node Name
+       \required-field
+       \type object-list
+       \object-list ConnectionNames
+  A4 , \field Steam Outlet Node Name
+       \required-field
+       \type object-list
+       \object-list ConnectionNames
+  N1 , \field Nominal Capacity
+       \autosizable
+       \units W
+       \minimum 0.0
+       \required-field
+  A5 ; \field Capacity Fraction Schedule
+       \note Schedule values are multiplied by Nominal Capacity for current capacity
+       \type object-list
+       \object-list ScheduleNames
```

Comment:
My gut is make DistrictHeating::capacityFractionSchedule and DistrictHeatingSteam::capacityFractionSchedule since both are new fields and I really hope we can reduce optionals as much as possible. Then I guess the question is do we make this required in the IDD? I think yes, but then we need to do version translation for the existing DistrictHeating type. I suppose the alternative would be to handle it in the API and return a default schedule if the osm does not define one, but I'm guessing that is not a popular option.

Reply:
I can make them non optional, in the new PR #4972 if we think that's better. VT isn't that complicated.